<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LMMS Theme Installer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="0">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="themeTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Themes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,3">
        <item row="0" column="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QWebView" name="themeDescriptionWebView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QListWidget" name="themeListWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="presetTab">
       <attribute name="title">
        <string>Presets</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
        <item row="0" column="0">
         <widget class="QListWidget" name="presetListWidget"/>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="margin">
            <number>1</number>
           </property>
           <item row="0" column="0">
            <widget class="QWebView" name="presetDescriptionWebView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="url">
              <url>
               <string>about:blank</string>
              </url>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionOptions">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
